docs/src/doc/getting-started/requirements.md

```diff
@@ -8,4 +16,4 @@ There is currently no plan to support the built-in Godot editor, using [IntelliJ
 We also recommend installing our [IntelliJ IDEA plugin](https://plugins.jetbrains.com/plugin/16505-godot-jvm) as it provides you with a lot of useful checks and helpers to detect errors during development.
 
 ## Godot Editor
-You can download our custom engine build and export templates from the latest [github release](https://github.com/utopia-rise/godot-kotlin-jvm/releases). If you need other modules you can also [build it yourself](../contribution/setup.md).
\ No newline at end of file
+You can download our custom engine build and export templates from the latest [github release](https://github.com/utopia-rise/godot-kotlin-jvm/releases). If you need other modules you can also [build it yourself](../contribution/setup.md).
```

Comment:
Am I blind? I can't seem to see the difference between the these particular lines. 🤔

Reply:
I think this is the new line at the end of file (which GitHub puts automatically when you edit a file because it's often helpful with diffs in the long run)